addons/calendar/models/calendar.py

```diff
@@ -698,8 +698,8 @@ def _get_date_formats(self):
             }
 
         # formats will be used for str{f,p}time() which do not support unicode in Python 2, coerce to str
-        format_date = pycompat.to_native(lang_params.get("date_format", '%B-%d-%Y'))
-        format_time = pycompat.to_native(lang_params.get("time_format", '%I-%M %p'))
+        format_date = pycompat.to_text(lang_params.get("date_format", '%B-%d-%Y'))
+        format_time = pycompat.to_text(lang_params.get("time_format", '%I-%M %p'))
```

Comment:
can probably remove the coercion call entirely: point was mostly to get an `str` in Python 2 (the data from a `Char` field is always unicode)